lib/apivore/swagger_checker.rb

```diff
@@ -42,10 +44,12 @@ def base_path
       @swagger.base_path
     end
 
-    attr_reader :swagger_path, :mappings, :swagger
+    attr_reader :swagger_path, :untested_mappings, :swagger
 
     private
 
+    attr_reader :mappings
```

Comment:
Keeps the api of the module small. Also makes it easy to refactor as it is now an implementation detail